### PR TITLE
NEWS: add 'prev-kv' feature for upcoming v3.0.11

### PR DESCRIPTION
**Pull request by gyuho**
@heyitsanthony @xiang90

### CODE REVIEW

**Comment by xiang90** [comment]
LGTM

File: NEWS [hunk 1]
```diff
@@ -1,3 +1,12 @@
+etcd v3.0.11 (2016-10-07)
+
```
Comment (heyitsanthony) [inline NEWS]: server supports it too?

**Comment by gyuho** [comment]
@heyitsanthony Just fixed. PTAL. Thanks.

**Comment by heyitsanthony** [comment]
lgtm

**Pull request merged**